compiler/damlc/tests/src/DA/Test/Packaging.hs

```diff
@@ -132,9 +132,9 @@ tests damlc repl davlDar = testGroup "Packaging"
             , "name: b"
             , "source: ."
             , "dependencies:"
-            , "  - daml-prim"
-            , "  - daml-stdlib"
             , "  - " <> aDar
+            , "build-options:"
+            , "- '--package=a-1.0'"
```

Comment:
This doesn’t seem right, why do I need to specify `a`? I explicitly depend on it in `build-options`.

Reply:
this is sorted now